dbms/src/Storages/Page/stress/workload/MainEntry.cpp

```diff
@@ -0,0 +1,68 @@
+// Copyright 2022 PingCAP, Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+#include <Storages/Page/stress/workload/PSStressEnv.h>
+#include <Storages/Page/stress/workload/PSWorkload.h>
+
+using namespace DB::PS::tests;
+
+int StressWorkload::mainEntry(int argc, char ** argv)
+{
+    {
+        // in order to trigger REGISTER_WORKLOAD
+        void _work_load_register_named_HeavyMemoryCostInGC();
+        void (*f)() = _work_load_register_named_HeavyMemoryCostInGC;
+        (void)f;
```

Comment:
Use `[[maybe_unused]]` to eliminate the warnings?

Reply:
`[[maybe_unused]]` is not exact here. It is unused. 😄

Reply:
Why do we need so many `(void)f`? Is it ok to have only one `(void)f` at the end?

Reply:
https://ci.pingcap.net/blue/organizations/jenkins/tiflash-build-common/detail/tiflash-build-common/5821/pipeline
error: Value stored to 'f' is never read [clang-analyzer-deadcode.DeadStores,-warnings-as-errors]